source/node-lib-qt-rss/RssFeed.cpp

```diff
@@ -28,6 +29,11 @@ void RssFeed::redrawGUI(const v8::FunctionCallbackInfo<v8::Value>& args) {
     emit getInstance().entriesChanged();
 }
 
+void RssFeed::refreshFeed() {
+    node::lib::Evaluate("emitRequest()");
+    while (node::lib::ProcessEvents()){}
```

Comment:
we could also call RunEventLoop() with a callback here to make it less blocking, but I think this is enough for our application. Thoughts?

Reply:
Changed it to tick Qt loop, can't hurt anyway.